app/services/calendar_service.rb

```diff
@@ -81,6 +82,10 @@ def memorial
       find(&:monday?)
   end
 
+  def juneteenth
```

Comment:
Can we keep with convention of the other methods and include a comment with the date / pattern?

```suggestion
  # June 19th
  def juneteenth
```

Arguably not super valuable, but at least consistent.